Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
```

Comment:
### [bootsnap](https://github.com/Shopify/bootsnap)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.4.5 → 1.18.3

[_[change-log](https://github.com/Shopify/bootsnap/blob/main/CHANGELOG.md), [source-code](https://github.com/Shopify/bootsnap), [gem-diff](https://my.diffend.io/gems/bootsnap/1.4.5/1.18.3)_]





<details>
<summary>Commits</summary>

A change of **329** commits. See the full changes on [the compare page](https://github.com/Shopify/bootsnap/compare/v1.4.5...v1.18.3).

These are the first 10 commits:
- (68bf00a) [Rework support for YAML symbolize_names and drop the Marshal…](https://github.com/Shopify/bootsnap/commit/68bf00ac6672555d96322648f5867c20edd1096a)
- (2376064) [Merge pull request #342 from Shopify/improve-yaml-cache-args](https://github.com/Shopify/bootsnap/commit/23760640ad5288bee07bd3c785f311d2408eb2d2)
- (9a2f1f3) [Release 1.6.0](https://github.com/Shopify/bootsnap/commit/9a2f1f3708e24de6de898f995f38053172e4c2c0)
- (53ace7f) [Expand paths with realpath in YAML cache](https://github.com/Shopify/bootsnap/commit/53ace7f580954e05bff2bd78734e0037b4e96a4b)
- (f52e16b) [Use GC.verify_compaction_references to make sure C reference…](https://github.com/Shopify/bootsnap/commit/f52e16b643893047981e63989cd5ba3abb315362)
- (b941bad) [Allow to turn caches off through environment variables](https://github.com/Shopify/bootsnap/commit/b941badd945e32e4c19cafbd90f9ae59844bc5dc)
- (2d45c5e) [Deprecate ActiveSupport::Dependencies integration](https://github.com/Shopify/bootsnap/commit/2d45c5e2303a2d27e58086d763e19d42341dd36d)
- (8c7845d) [Merge pull request #344 from Shopify/deprecate-activesupport](https://github.com/Shopify/bootsnap/commit/8c7845d864b4a9edd19c7bd102607b16420c7c86)
- (0a1283d) [Deprecate disable_trace option](https://github.com/Shopify/bootsnap/commit/0a1283d0c7de99e527b81d25b4bea1114038932b)
- (9d6ff0e) [Allow to control the behavior of `require 'bootsnap/setup'` …](https://github.com/Shopify/bootsnap/commit/9d6ff0e94ec304fa3e79360e663a98493f4ce7b9)


</details>